modules/apps/27-interchain-accounts/types/validate.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"regexp"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// DefaultMaxAddrLength defines the default maximum character length used in validation of addresses
+var DefaultMaxAddrLength = 64
```

Comment:
I think we maybe don't need to check for the length of the address here. We may want to leave this flexible for non cosmos sdk account addresses. wdyt @colin-axner

Reply:
Is there an upper limit for Version string characters already?

Reply:
I picked these numbers completely arbitrarily. I think it would be good to still enforce some kind of max length here. If 64 isn't enough we can increase to 128/256

Reply:
Makes sense then :+1: